resources/views/correspondence/create.blade.php

```diff
@@ -0,0 +1,114 @@
+            </div>
+        </section>
+
+        <section class="form">
+            <form action="{{ route('correspondence.save') }}" method="POST">
+                @csrf
+                <div class="row">
+                    <div class="col-5">
+                        <label for="recipient">* Destinatário</label>
+                        <input type="text" name="recipient" id="recipient" autofocus class="input-form" required placeholder="Nome">
+                        @error('recipient')
+                            <div class="alert-error"><i class="fas fa-exclamation-triangle"></i> {{ $message }} </div>
+                        @enderror
+                    </div>
+                </div>
+                <div class="row">
+                        <div class="col-4">
+                            <label for="street">* Logradouro</label>
+                            <input type="text" name="street" id="street" class="input-form" required placeholder="Logradouro">
+                            @error('street')
+                                <div class="alert-error"><i class="fas fa-exclamation-triangle"></i> {{ $message }}</div>
+                            @enderror
+                        </div>
+                        <div class="col-1">
+                            <label for="number">* Número</label>
+                            <input type="number" name="number" id="number" class="input-form" min="0" required placeholder="Número">
+                            @error('number')
+                                <div class="alert-error"><i class="fas fa-exclamation-triangle"></i> {{ $message }} </div>
+                            @enderror
+                        </div>
+                    </div>
+                    <div class="row">
+                        <div class="col-2">
+                            <label for="neighborhood">* Bairro</label>
+                            <input type="text" name="neighborhood" id="neighborhood" class="input-form" required placeholder="Bairro">
+                            @error('neighborhood')
+                                <div class="alert-error"><i class="fas fa-exclamation-triangle"></i> {{ $message }} </div>
+                            @enderror
+                        </div>
+                        <div class="col-2">
+                            <label for="city">* Cidade</label>
+                            <input type="text" name="city" id="city" class="input-form" required placeholder="Cidade">
+                            @error('city')
+                                <div class="alert-error"><i class="fas fa-exclamation-triangle"></i> {{ $message }} </div>
+                            @enderror
+                        </div>
+                        <div class="col-1">
+                            <label for="uf">* UF</label>
+                            <select id="if" name="uf" class="input-form" placeholder="teste" >
```

Comment:
```suggestion
                            <select id="uf" name="uf" class="input-form">
```
Não seria melhor pegar as options e colocar dentro de um array e fazer um foreach?